lib/ITip/Broker.php

```diff
@@ -404,8 +404,12 @@ protected function processMessageReply(Message $itipMessage, VCalendar $existing
 
                 $newObject = $recurrenceIterator->getEventObject();
                 $recurrenceIterator->next();
-
-                if (isset($newObject->{'RECURRENCE-ID'}) && $newObject->{'RECURRENCE-ID'}->getValue()===$recurId) {
+                if (isset($newObject->{'RECURRENCE-ID'}) && $newObject->{'RECURRENCE-ID'}->getDateTime()->getTimestamp() === $recurId) {
+                    $found = true;
+                } elseif ($recurrenceIterator->key() === 1 && $newObject->DTSTART->getDateTime()->getTimestamp() === $recurId) {
```

Comment:
Could you explain this line? `key()` in particular strikes as a bit odd. Wouldn't you want to refer to the first event? In addition to that, there's no guarantee that the first event is the master event. It's possible to make a recurrence and create an exception for the second instance, and set the DTSTART for that _before_ the master event.

Maybe I simply don't fully understand this change =)